sublime_jedi/tooltips/markdown.py

```diff
@@ -53,17 +53,20 @@ def _prepare_signature(self, docstring):
         """
         pattern = (
             '(?x)'
-            '^([\w\. \t]+\.[ \t]*)?'        # path
-            '(\w+)'                         # function / object
-            '[ \t]*(\([^\)]*\))'            # arguments
-            '(\s*->\s*.*$)?'                # annotation (rest of line)
+            '^([\w\. \t]+\.[ \t]*)?'    # path
+            '(\w+)'                     # function / object
+            '[ \t]*(\([^\)]*\))'        # arguments
+            '(?:'
+            '(\s*->\s*.*?)'             # annotation: -> Type
+            '(--|$)'                    # inline comment: -- blabla
+            ')?'
```

Comment:
You stick too much to markdown :-) The idea was just to push the docstring through a markdown compiler to see whether it can highlight some pieces as it is part of mdpopups plugin anyway. Python docstring in general don't use markdown at all. 

I personally use _Google_ style docstrings as they seem much clearer than normal rst. So I am basically fine with highlighted signature and plain text docstring right now. I didn't intend to start to create a couple of docstring interpreters to format all kinds of docstrings so far. Maybe this is something which could be done in a second step. I don't have time to do it at the moment.